README.md

```diff
@@ -90,6 +95,11 @@ Flags:
       --upload_region                            The region the metrics-agent will upload data to. (default `us-west-2`)
       --custom_s3_bucket string                  A custom S3 bucket the metrics-agent will upload data to. - Optional
       --custom_s3_region                         The AWS region that the custom s3 bucket is created.
+      --custom_azure_blob_container_name string  A custom Azure blob container name the metrics-agent will upload data to. - Optional
+      --custom_azure_blob_url                    A custom Azure blob url metrics-agent uses to create the Azure client.
+      --custom_azure_tenant_id                   A custom Azure Service principal tenant id the metrics-agent uses to create the Azure client.
```

Comment:
this documentation I think is a bit messy already but shouldn't all these --custom_azure flags have `- Optional`

Reply:
i'll fix them all while here